tests/ignite/distributed/test_auto.py

```diff
@@ -25,14 +25,18 @@ def _test_auto_dataloader(ws, nproc, sampler_name=None, dl_type=DataLoader):
 
     # Test auto_dataloader
     assert idist.get_world_size() == ws
-    dataloader = auto_dataloader(data, batch_size=10, num_workers=2, sampler=sampler, shuffle=sampler is None)
+    BATCH_SIZE = 10
```

Comment:
Maybe the test case with batch size smaller than world size is needed to

Reply:
@vfdev-5 that sounds definitely better than this